content/v1.11/getting-started/provider-aws-part-2.md

```diff
@@ -515,20 +515,30 @@ required:
 ```
 
 ### Enable claims to the API
-Allow a _claim_ to use this _XRD_ by defining the _claim_ API endpoint under the _XRD_ 
-{{<hover label="XRDclaim" line="4">}}spec{{< /hover >}}.
+Tell this _XRD_ to offer a _claim_ by defining the _claim_ API endpoint under
+the _XRD_ {{<hover label="XRDclaim" line="4">}}spec{{< /hover >}}.
```

Comment:
I'm not tied to this phrasing, but "allow a claim to use this XRD" doesn't seem accurate to me. It misses the context that the XRD is defining a type of claim into existence.

More broadly I think it's worth being aware of the different types of relationships we have and using more specific words than "uses". For example:

* XRD _defines_ XR
* XRD _defines_ XRC
* XRC _claims_ XR
* Composition _specifies_ MRs (maybe _templates_? I'm a little resistant to that word)
* XR _uses_ Composition (in this case "uses" does seem the most accurate to me)
* XR _composes_ MRs

Reply:
The point in the original text was to say "you can't have a claim if it's not defined in the XRD".  Maybe my mental model is wrong but I view an XRD as almost being an element with permissions. You can't access it via a claim unless the XRD _allows_ it.  

I'm open to changing it but "offers" doesn't feel like the right word here, that feels like a pub/sub thing, as if the XRD is producing an artifact (maybe it is, but it's not directly relevant to this part of the doc). I'm coming up blank on what to use an alternative. 
Tell this XRD to
 - allow a claim
 - allow the creation of claims 
 - create a claim type (this doesn't seem right) 
 - admit claims....something....

Reply:
Let me try a dumb analogy - I'm at the beach making a sandcastle. The previous phrasing reads to me as equivalent to "Allow a sandcastle to use Nic to create it".

Where _offer_ comes from is the idea that:

* The platform team is defining a new API type.
* By default is not available to developers to claim. As an XR (only) it's private to the platform team.
* The platform team may choose to offer the API type to their end users. They do this by offering a claim.
* Offering a claim is done by specifying `spec.claimNames` on an XRD.

My preference would be to stick with the concept of "offering a claim". This is an established concept that permeated our previous documentation and shows up in Kubernetes events, errors, etc pertaining to the concept. e.g. https://github.com/search?q=org%3Acrossplane+offer&type=code

Reply:
```suggestion
Tell this _XRD_ to offer developers a _claim_ by defining the _claim_ API endpoint under
the _XRD_ {{<hover label="XRDclaim" line="4">}}spec{{< /hover >}}.
```

The offer idea breakdown is super helpful. 

Let's just specific _who_ the claim is being offered to. It reads as if it's a Kubernets object offering something to another object. Can we just add "developers" to it's clear who the claim is being offered to? I'm sorry if this is too nit picky, I'm trying to write for someone like me who has little background knowledge on Crossplane and I'm trying to hammer out all of the assumptions or implied meanings.